tests/ops/qubit/test_parametric_ops.py

```diff
@@ -171,7 +171,13 @@ def circuit(phase, dim):
             return qml.state()
 
         def _get_expected_state(phase, dim, size):
-            return np.array([np.exp(1j * phase) if i < dim else 1.0 for i in range(size)]) * 1 / 2
+            return (
```

Comment:
How this is computed? If computed by hand or it is a trivial solution, might add a comment to clarify.

Reply:
Its trivial, the result is by definition of the operator